packages/block-library/src/buttons/editor.scss

```diff
@@ -37,12 +37,7 @@ $blocks-block__margin: 0.5em;
 	//
 	// Disable the stylelint rule, otherwise this selector is ugly!
 	/* stylelint-disable indentation */
-	&:not(
-		.is-content-justification-space-between,
-		.is-content-justification-right,
-		.is-content-justification-left,
-		.is-content-justification-center
-	) .wp-block[data-align="center"] {
```

Comment:
This is an interesting one, because this is valid CSS and supported by browsers, but the spec this is part of is technically a working draft, which is why the validator has flagged it.

The 'note' part here mentions it:
https://drafts.csswg.org/selectors/#negation

Reply:
The way `:not()` contributes to specificity is a little strange: the specificity is consistently the value of the most specific selector in the list.

For the existing code, this results in the `:not()` selector contributing one class to the specificity. Bringing the specificity of the generated selector to `0, 0, 4, 0`.

In the proposed code, each instance of the `:not()` selector contributes one class. Bring the specificity of the generated selector to `0, 0, 7, 0`.

Given the code works in all browsers WP and Gutenberg supports, I'd suggest keeping the code as currently published. The reported validator issue isn't a practical issue.

Reply:
I think I agree with leaving this one as is.

@mukeshpanchal27 Do you agree, and are you able to update this PR to address this?

Reply:
@desrosj PR updated as requested.